lib/pharos/core-ext/colorize.rb

```diff
@@ -18,7 +25,7 @@ def self.enabled?
       end
 
       refine String do
-        Pastel::ANSI::ATTRIBUTES.each_key do |meth|
+        ATTRIBUTES.each do |meth|
```

Comment:
Is this really needed?

Reply:
Yes but the line below isn't anymore as `underscore` can be removed from the ATTRIBUTES array.

Alternatives would be to define each method delegator manually, add a `method_missing` delegator or going back to regular pastel.

Reply:
So what do we gain by copypasting Pastel attributes here?

Reply:
We get them without loading pastel which loads `tty-color` and `equatable`.